### PR TITLE
Add index.html

### PR DESCRIPTION
**Pull request by wangkuiyi**
So we can add a git submodule in https://github.com/sql-machine-learning/sql-machine-learning.github.io to enable redict from sqlflow.org/playground to https://github.com/sql-machine-learning/playground

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+Our solution is to build a Website https://sqlflow.org, whose page
+http://sqlflow.org/sqlflow/index.html contains some meta information
+readable by the Go toolchain, so that the `go get` command knows that
+`import sqlflow.org/sqlflow` actually redirects to `import
+github.com/sql-machine-larning/sqlflow `.
+
+We build the Website by the following steps.
+
+1. We configure https://sqlflow.org pointing to
+http://sql-machine-learning.github.io, which is a Github Page Web site
```
Comment (terrytangyuan) [inline index.html]: Github -> GitHub

File: index.html [hunk 2]
```diff
@@ -0,0 +1,42 @@
+<!-- Please don't delete this file!
+
+This file is generated by https://github.com/bramp/goredirects. We
+need it because github.com/sql-machine-larning/sqlflow is too long a
+Go import name.  We want it to be sqlflow.org/sqlflow.
+
+Our solution is to build a Website https://sqlflow.org, whose page
```
Comment (terrytangyuan) [inline index.html:7]: Website -> website

File: index.html [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+http://sqlflow.org/sqlflow/index.html contains some meta information
+readable by the Go toolchain, so that the `go get` command knows that
+`import sqlflow.org/sqlflow` actually redirects to `import
+github.com/sql-machine-larning/sqlflow `.
+
+We build the Website by the following steps.
```
Comment (terrytangyuan) [inline index.html]: Website -> website

**Review by tonyyang-svail — DISMISSED** [review]
LGTM

**Review by terrytangyuan — APPROVED** [review]

**Pull request merged**